### PR TITLE
Add Acronym Identification Dataset

### PR DESCRIPTION
**Pull request by abhishekkrthakur**

### CODE REVIEW

File: datasets/acronym_identification/README.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+### Languages
+
+[More Information Needed]
+
+## Dataset Structure
+
+### Data Instances
+
+```
+DatasetDict({
```
Comment (lhoestq) [inline datasets/acronym_identification/README.md:72]: In this section we expect to see an example of data sample, as the one you added in the Data Field section. Can you move it in this section instead please ?

In the Data Field section you can add a brief description of each one of the data fields id, tokens labels, and also explain the BIO format used for labels

File: datasets/acronym_identification/README.md [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+  'traditionally',
+  'been',
+  'given',
+  'many',
+  'different',
+  'names',
+  '.']}
+```
+
+Please note that in test set sentence only id, tokens are available. labels can be ignored for test set.
```
Comment (lhoestq) [inline datasets/acronym_identification/README.md:115]: You can add that the missing labels of the test set are all replaced with the label 'O'

**Review by lhoestq — COMMENTED** [review]
Really cool thanks !

I left a few comments about the dataset card, the rest is all good :)

**Comment by abhishekkrthakur** [comment]
fixed @lhoestq

**Review by lhoestq — APPROVED** [review]
Thanks !

**Pull request merged**